noir/scripts/test_js_packages.sh

```diff
@@ -14,6 +14,7 @@ cargo build --release
 export PATH="${PATH}:/usr/src/noir/noir-repo/target/release/"
 
 yarn --immutable
+yarn version::stables
```

Comment:
`yarn workspace docs version::stables`

alternatively update the root package.json to have
```
    "build": "yarn workspaces foreach -vtp --exclude \"{docs,@noir-lang/root}\" run build",
```

Reply:
We may want a separate CI stage added afterwards just to ensure that the docs still build without blocking the rest of CI however.

Reply:
yup that sounds good!